src/main/java/uk/gov/companieshouse/controller/DissolutionController.java

```diff
@@ -14,8 +14,8 @@
 import uk.gov.companieshouse.exception.DirectorNotPendingApprovalException;
 import uk.gov.companieshouse.exception.DissolutionNotFoundException;
 import uk.gov.companieshouse.exception.UnauthorisedException;
-import uk.gov.companieshouse.model.dto.*;
-import uk.gov.companieshouse.service.DissolutionService;
+import uk.gov.companieshouse.model.dto.dissolution.*;
```

Comment:
Think IntelliJ automatically does this which is annoying. But we shouldn't use wildcards for imports. There's a few other classes where this is happening as well.

Reply:
You can change this in options. I just did.